tools-edge/oas-checker/README.md

```diff
@@ -0,0 +1,76 @@
+# OpenAPI v3 Checker
+
+This tool checks the endpoint provided using OpenAPI v3 specification
+using a set of rules capable of security and semantic checks.
+
+This tool is based on stoplight/spectral.
+
+**Tool type**: action
+
+## Tool description
+
+Almost all APIs are published using the OpenAPI specification. In order to
+provide a good API definition some best pactices should be followed.
+
+A linter is a tool that analizes files written in a specific language looking
+for incorrect constructions and suspicious or incorrect code.
+
+This tool accepts an endpoint definition written using OpenAPI v3 as input
+(provided as an argument or read from standard input) and will output the same
+definition, if no erros are encounterd, or the list of errors detected.
+
+In case of error, a code of 1 is returned, 0 otherwise.
+
+## Quick start
+
+Install the tool:
+
+```console
+$ acp install oas-checker
+[*] Fetching Docker image for tool 'oas-check'
+
+    Using default tag: latest
+    latest: Pulling from stoplight/spectral
+    aad63a933944: Already exists
+    dc24e89b59ec: Already existshttp://my-company.com/api/entry-point-v3.yml
+    810779e0b9c3: Already exists
+    ...
+    Status: Downloaded newer image for bbvalabs/oas-check:latest
+    docker.io/bbvalabs/oas-check:latest
```

Comment:
@cr0hn do we need this here? The image is actually diferent :)

Reply:
This is an example of the tool execution when running the tool from the package manager (plus a typo in the original file you copied from that has to be fixed in both files ;-)

Don't need to correspond to the current docker image, but we think is good to show new users an example of what they're goin to see when executing the tool.